tools/n64tool.c

```diff
@@ -245,6 +253,9 @@ int main(int argc, char *argv[])
 				fprintf(stderr, "WARNING: The output should have a '.z64' file extension\n");
 			}
 
+			tmp_output = malloc(output_len + 4 + 1);

```

Comment:
What's the rationale for this? If these numbers are meaningful, it might be better for readability to declare them as constants (or at least throw in a comment explaining what's happening here).

Reply:
Oh, I get it: this is for `.tmp` -- should have read a bit further into it. 😄

Reply:
Suggestion: Add `*.tmp`  to `.gitignore` under `## Build files`

Reply:
Also, add `*.tmp` to the Makefile `clean` targets for all the examples.

Alternatively, consider using ISO C `tmpfile` to manage the temporary file and copy it to output at the end.

Reply:
If you don't decide to use `mkstemp`: consider changing this line to `asprintf(tmp_output, "%s.tmp", output)` for better readability

Reply:
> Suggestion: Add *.tmp to .gitignore under ## Build files

It's already in there, albeit in a different section

> Also, add *.tmp to the Makefile clean targets for all the examples.

Do we really need it, given that I also registered deletion in `atexit`? I would say that it should be a very rare occurrence (eg: a segfault). Also, they're ignored anyhow, it's not that they confuse the Makefile by creating a false dependency.

> If you don't decide to use mkstemp: consider changing this line to asprintf(tmp_output, "%s.tmp", output) for better readability

Done.